docs/src/SUMMARY.md

```diff
@@ -41,6 +29,7 @@
     - [Actors](./explore/design/actors.md)
     - [Proof of Stake system](./explore/design/pos.md)
     - [Testnet setup](./explore/design/testnet-setup.md)
+    - [Testnet launch procedure](./explore/design/testnet-launch-procedure/README.md)
```

Comment:
These two testnet guides feel out of place in the 'Design' section but not sure where they could be better located right now. Maybe we eventually would want an 'Infra' section or something similar

Reply:
True yeah. For now the explore section is very free form so should be okay until we organize it better. Thanks for reviewing!